nilearn/_utils/fixes/scikit_learn_validation.py

```diff
@@ -0,0 +1,400 @@
+# scikit-learn v0.17
```

Comment:
This is exactly the purpose of this file: bring a feature available in sklearn 0.17 to previous versions. For me, this is self-explanatory. Maybe just say that this file is from sklearn.utils.validation.